### DISCUSSION TITLE
You should turn on sponsoring for this project, so we can all give you money

### DISCUSSION THREAD

**Discussion by jeremiahsherrill** [wdelfuego/nova-calendar#55] [Ideas & Feature Requests]
Turn on github sponsoring, so you can get sponsors for your hard work

**Comment by wdelfuego** [comment]
Ha, that's very kind of you. I've submitted an application to GitHub, awaiting approval.

Sponsoring won't be necessary for anyone but I guess it can't hurt to offer the option. Especially corporate sponsors would be awesome, I intend to offer some valuable support in return.

Thanks for your kind words 🤗

**Comment by wdelfuego** [comment]
I was approved, anyone interested can now sponsor this project. Thanks for the push, @jeremiahsherrill 😄!

Let me know if you have ideas about tiers and rewards, all feedback is welcome.
